Makefile

```diff
@@ -2,6 +2,7 @@
 .PHONY: help install install-scripts install-conf install-systemd uninstall
 
 ### Macros ###
+DEFAULT_BRANCH = master
```

Comment:
I think we can get this on-the-fly with

```bash
basename $(git symbolic-ref refs/remotes/origin/HEAD)
```

as this is subject to change. I've been thinking about renaming `master` to `main` on old repos as well but not yet made it.